blocks/tabs/tabs.css

```diff
@@ -6,11 +6,24 @@
   flex-direction: row;
 }
 
+.full-view {
+  margin-top: -80px;
+}
+
+.full-view .tabs-list {
+  border-bottom: solid 1px var(--button-secondary-text-color) ;
+}
+
 @media (width >= 1280px) { 
   .tabs .tabs-list {
     justify-content: flex-end; /* This shifts the tab buttons to the right on large screens */
   }
-}
+
+  .full-view .tabs-list{
+    justify-content: flex-start;
+    border-bottom: solid 1px var(--button-secondary-text-color) ;
```

Comment:
You dont need to set border bottom for desktop, as its already set for all viewports on line no. 13-15